armory/eval/evaluator.py

```diff
@@ -130,20 +133,30 @@ def run(self, interactive=False, jupyter=False, host_port=8888) -> None:
     def _run_config(self, runner) -> None:
         logger.info(bold(red("Running evaluation script")))
         runner.exec_cmd(
-            f"python -m {self.config['evaluation']['eval_file']} {self.unix_config_path}"
+            f"python -m {self.config['evaluation']['eval_file']} {self.docker_config_path}"
         )
 
     def _run_interactive_bash(self, runner) -> None:
+        if os.name != "nt":
+            user_id = os.getuid()
+            group_id = os.getgid()
+        else:
+            user_id = 0
+            group_id = 0
         lines = [
             "Container ready for interactive use.",
             bold(
                 "*** In a new terminal, run the following to attach to the container:"
             ),
-            bold(red(f"    docker exec -itu0 {runner.docker_container.short_id} bash")),
+            bold(
+                red(
+                    f"    docker exec -it -u {user_id}:{group_id} {runner.docker_container.short_id} bash"
```

Comment:
The same user message should be on line 163

Reply:
Good catch thanks!. Fixed.